components/proxy/Dockerfile

```diff
@@ -2,55 +2,11 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM openresty/openresty:1.19.3.1-3-alpine
-
-ENV TRIGGER_REBUILD 1
+FROM aledbf/caddy-http2:0.5
```

Comment:
Can we build this caddy image as part of the Gitpod repo, or is this a fork of caddy?

Reply:
This image is a fork of Caddy with the updated packages https://github.com/aledbf/caddy/commit/68fbb8eaa803b2b867fd79b230acea625e4619f4
I can do the same thing in the gitpod org, forking caddy and adding the commit, or maintain the code in my fork until the fix land in Caddy.